examples/servlet/src/main/java/dev/openfga/example/DocumentService.java

```diff
@@ -0,0 +1,75 @@
+package dev.openfga.example;
+
+import dev.openfga.sdk.api.client.OpenFgaClient;
+import dev.openfga.sdk.api.client.model.ClientTupleKey;
+import dev.openfga.sdk.api.client.model.ClientWriteRequest;
+import dev.openfga.sdk.errors.FgaInvalidParameterException;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Service;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+
+@Service
+public class DocumentService {
+
+    private final Set<Document> documentSet = new HashSet<>();
+    private final OpenFgaClient fgaClient;
+
+    public DocumentService(OpenFgaClient fgaClient) {
+        this.fgaClient = fgaClient;
+    }
+
+    /**
+     * Return the requested document, if it exists, only if the user has the
+     * required FGA relationship.
+     *
+     * @param id The ID of the document to get.
+     * @return the document.
+     */
```

Comment:
Can we explain why we are hardcoding the user id here? Users wouldn't be doing this in prod.

A follow-up, if we can use JWT and provide a two signed jwts people can test with with two different users, so we explain how the user id is retrieved

Reply:
Yes, we should at a minimum add explanation here. 

I do like the idea of using a JWT to better illustrate more real-life scenario. When I worked on the prototype, we had an example that showed an application that would perform a client credentials grant to obtain a JWT, and then it would call another application that was a resource server secured with JWT and FGA PreAuthorize ([example](https://github.com/Crain-32/fga-spring-examples/tree/d793c2d0ac01e6acda75c0535bec62daf8c9d699?tab=readme-ov-file#client-credentials-sample)). It required using the okta-spring-boot-starter to configure the token exchange and resource server, and maybe seems a bit too much for an example here?

Maybe there's a better/simpler way to demonstrate using a JWT, let me think on it, and chime in if you have any ideas.

Reply:
I updated the sample app to use a (hard-coded) JWT to create an Authentication, from which the FGA check will obtain the principal name. This enabled me to remove the hard-coded user ID in the FGA check.